### PR TITLE
docs: Update README files for all of the Dart packages

### PR DESCRIPTION
**Pull request by xavierchanth**
**- What I did**

Updated the various dart package readmes with the latest relevant information:
- `No Ports` -> `NoPorts`
- noports_core points to npt_flutter instead of sshnp_flutter as it's app example.
- marked sshnp_flutter as unmaintained and added suggested alternatives
- Updated sshnoports a little bit, it could use more work, but I'm timeboxing it here.

**- How I did it**

**- How to verify it**

**- Description for the changelog**
docs: Update README files for all of the Dart packages

### CODE REVIEW

File: packages/dart/sshnoports/README.md [hunk 1]
```diff
@@ -4,13 +4,12 @@
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/atsign-foundation/noports/badge)](https://api.securityscorecards.dev/projects/github.com/atsign-foundation/noports)
 [![SLSA 3](https://slsa.dev/images/gh-badge-level3.svg)](https://slsa.dev)
 
-# SSH! No ports
+# NoPorts Cli Binaries
```
Comment (cpswan) [inline packages/dart/sshnoports/README.md]: s/Cli/CLI/

File: packages/dart/sshnoports/README.md [hunk 2]
```diff
@@ -4,13 +4,12 @@
...
 
-ssh no ports provides a way to ssh to a remote linux host/device without that
-device or the client having any open ports (not even 22) on external interfaces. All
-network connectivity is out bound and there is no need to know the IP
-address the device has been given. As long as the device and client has an IP address (public or private 1918),
-DNS and Internet access, you will be able to connect to it.
+This is the source code for the NoPorts cli binaries.
```
Comment (cpswan) [inline packages/dart/sshnoports/README.md]: s/cli/Command Line Interface (CLI)/

**Review by cpswan — CHANGES_REQUESTED** [review]
Mostly LGTM but can be please use correct capitalisation for CLI.

File: packages/dart/sshnoports/README.md (hunk 2 above)
Comment (xavierchanth) [reply packages/dart/sshnoports/README.md]: fixed in [0e23e54](https://github.com/atsign-foundation/noports/pull/1613/commits/0e23e54562176433076875067b0a3f7c94563856)

File: packages/dart/sshnoports/README.md (hunk 1 above)
Comment (xavierchanth) [reply packages/dart/sshnoports/README.md]: fixed in [0e23e54](https://github.com/atsign-foundation/noports/pull/1613/commits/0e23e54562176433076875067b0a3f7c94563856)

**Review by cpswan — APPROVED** [review]

**Pull request merged**